### ISSUE TITLE
Have the edit actions on the collection been reviewed?

### ISSUE THREAD

**Issue by cam156** [projecthydra/sufia#2409]
There are both Remove from collection and delete.  Remove from collection does not have an icon, which leads me to believe the options have not really been looked at.

![screen shot 2016-08-08 at 2 16 01 pm](https://cloud.githubusercontent.com/assets/1599081/17490812/bb78271a-5d72-11e6-92d0-f66ba0ec2928.png)

We (PSU) had at one point thought deleting an item from the repository on the collection edit screen might be confusing to the user.  That said if someone has done this thoughtfully we can close the issue.

**Comment by mjgiarlo** [comment]
I'm inclined to agree with you, @cam156.

Thoughts, @projecthydra/sufia-code-reviewers @projecthydra/sufia-ui-ux-advisors?

**Comment by mtribone** [comment]
Based on how much focus was on the creation of a work and your observation, I would say that the actions for works *in* collections needs to be reviewed.

**Comment by newmanld** [comment]
Agreeing with @cam156, ‘Delete work’ seems out of place on a collection edit screen. So does ‘Transfer Ownership of Work.’  Removing a work from a Collection is presumably needed, but with an icon added.  Edit Work seems safe.

But I’m not sure I have the entire context - are we talking about the catalog or a works dashboard, in which case ‘Delete Work’ and ‘Transfer Ownership of Work’ need to appear somewhere, just perhaps not mixed with collection actions?  (And in that case, where is ‘Add to Collection’ as an action?)

Or are we talking about a drop down menu that only appears when listing works already members of a collection?   If the latter, I lean toward removing ‘Delete Work’, and ‘Transfer Ownership of Work’ but would keep ‘Edit Work’ as a convenient route for the user, and assuming that after selecting ‘Edit Work’ they would have the option to either delete the work or transfer ownership of the work.

**Comment by cam156** [comment]
@newmanld We are talking about the list of actions the user gets on the collection Edit page, not on any of the work listings.  I agree removing both Delete and Transfer would make sense.  Thanks for the feedback!

**Comment by cam156** [comment]
Icon for remove should be an X.  The  delete option should not appear in the list.

**Comment by mjgiarlo** [comment]
@cam156 are y'all planning to fix this bug? I'm trying to decide if this should be slotted into Sufia 7.3.0 or get punted over to Hyrax (or some other option).

**Comment by cam156** [comment]
@mjgiarlo we are no longer fixing anything in Sufia and are pinned to 7.2-migration branch until we can get a stable product to deploy to production.  At some point in the far future we might get back to this in sufia or what ever incarnation of the code exists at that point.

**Comment by mjgiarlo** [comment]
@cam156 Yup, I know the drill. No problem. Do you have a fix for it in ScholarSphere, or are you likely to have one soon? I asked because of the `PSU Backlog` label on the issue. I'm trying to decide whether this belongs in 7.3.0 or can wait.

**Comment by mjgiarlo** [comment]
In response to the original question from @cam156:
> We (PSU) had at one point thought deleting an item from the repository on the collection edit screen might be confusing to the user. That said if someone has done this thoughtfully we can close the issue.

If we discover work-related actions are confusing in usability testing, that would be good to know. 'Til then, I'm reluctant to put effort into removing items from this menu. That said, yes, it's inconsistent to have one action without an icon. I suggest we either write up a new bug ticket for 7.3.0 to add an icon to this action, or reframe this ticket to be that. Thoughts?

**Comment by jcoyne** [comment]
@cam156 @mtribone Is this still an issue? It looks like the scholarsphere ticket was closed.

**Comment by cam156** [comment]
@jcoyne looks like that got closed by mistake in ScholarSphere, but we are not currently putting changes into Sufia, so it being closed in SS would not close it in Sufia.

**Comment by mtribone** [comment]
I think it was already resolved here: https://github.com/projecthydra/sufia/commit/798f09257137678bdd6fc38c3b52080de2778bc6

**Comment by cam156** [comment]
@mtribone that is all show actions, not the edit actions.  The delete is in the edit actions.
